services/common/src/redux/selectors/projectSelectors.ts

```diff
@@ -21,22 +21,38 @@ const formatProjectSummaryParty = (party): IParty => {
   if (!party?.party_guid) {
     return party;
   }
-  console.log(
-    party,
-    party.job_title_code,
-    party.first_name,
-    party.party_name,
-    party.address.length
-  );
+
   return { ...party, address: party.address[0] };
 };
 
+const formatProjectContact = (contacts): IProjectContact[] => {
+  if (!contacts) {
+    return contacts;
+  }
+
+  const primaryContact = contacts.filter((contact) => contact.is_primary);
+  const secondaryContacts = contacts.filter((contact) => !contact.is_primary);
+  const formattedContacts = [...primaryContact, ...secondaryContacts].map((contact) => {
+    const address = contact?.address;
+    return { ...contact, address: address?.length > 0 ? address[0] : null };
```

Comment:
for lines 36 and 37, could just set `address = contact?.address[0]` and skip the ternary in 37.

I do wonder if you _need_ to separate out primary and secondary contacts, since they don't seem to be modified differently.

Reply:
The ternary is changed now to the suggestion. 

Ah the reason I separated out the primary and secondary was in the project contacts section on the front end sometimes the primary contact data ends up being displayed as a secondary contact and a secondary contact ends up being displayed as the primary contact.